prime-router/src/main/kotlin/azure/BlobAccess.kt

```diff
@@ -115,6 +118,13 @@ class BlobAccess() : Logging {
             return System.getenv(blobConnEnvVar)
         }
 
+        /**
+         * Obtain the download retry value from the given environment.
+         */
+        fun getBlobDownloadRetry(blobDownloadRetryVar: String = defaultBlobDownloadRetryVar): Int {
```

Comment:
I wonder if we need to provide a default when the env var is not set?

Reply:
Yep, that makes sense to me out of an abundance of caution

Reply:
This has been resolved 👌 and will also handle cases where the env var is garbage.